shell/platform/android/io/flutter/view/ResourceExtractor.java

```diff
@@ -76,14 +88,55 @@ protected Void doInBackground(Void... unused) {
 
             return null;
         }
+
+
+        /// Returns true if successfully unpacked APK resources,
+        /// otherwise deletes all resources and returns false.
+        private boolean extractAPK(File dataDir) {
```

Comment:
Is this a long-running method? If so, should it be executed on the `@UiThread` or `@WorkerThread`?

Reply:
@sbaranov would know.  I'm going to imagine it's long running.

Reply:
Actually, this is only called from `doInBackground`, which I believe should mean it's fine to annotate this with `@WorkerThread`

Reply:
Yeah that sounds good.